### PR TITLE
feat(dataset-view): layout commit list items

### PR DESCRIPTION
**Pull request by chriswhong**
Basic layout for items the history tab

Qri_Desktop

### CODE REVIEW

File: app/components/DatasetSidebar.tsx [hunk 1]
```diff
@@ -12,6 +12,47 @@ const FileRow: React.SFC<FileRowProps> = (props) => {
   )
 }
 
+interface HistoryListItemProps {
+  commitTitle: string
+  avatarUrl: string
+  userTimeMessage: string
+}
+
+const HistoryListItem: React.SFC<HistoryListItemProps> = (props) => {
```
Comment (ramfox) [inline app/components/DatasetSidebar.tsx]: Should be `React.FunctionComponent`

File: app/scss/_dataset-container.scss [hunk 2]
```diff
@@ -133,6 +133,7 @@
     width: 100%;
```
Comment (ramfox) [inline app/scss/_dataset-container.scss]: Didn't we change this to "_dataset.scss" and move the previous "_dataset.scss" to "_dataset-old.scss"?

Comment (ramfox) [reply app/scss/_dataset-container.scss]: What do you think about moving the `.file-row` and `.history-list-items` stuff into it's own scss file. I think I'm liking the "one file one component" structure. Thoughts?

Comment (chriswhong) [reply app/scss/_dataset-container.scss]: I re-did the list-item stuff just now on another branch so it's cleaner and more centralized.  I think it's fine in `dataset.scss` as it probably won't be used outside of this view, but is shared by more than one child component

**Pull request merged**